From a54cb981ceecfd323e667f659a66032c7ae102a2 Mon Sep 17 00:00:00 2001
From: Evgueni Souleimanov <esoule@100500.ca>
Date: Tue, 18 Aug 2015 17:32:14 +0200
Subject: [PATCH] initscript: Make NetworkManager process /etc/sysctl.d/*

When NetworkManager is started or restarted, it applies /etc/sysctl.conf when
printing "Setting network parameters...". It does not apply settings from files
in /etc/sysctl.d/*. I have an unmodified file /etc/sysctl.conf and override
some settings in /etc/sysctl.d/ and NetworkManager does not apply my overrides.

https://bugzilla.redhat.com/show_bug.cgi?id=1213118
---
 initscript/RedHat/NetworkManager.in | 2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/initscript/RedHat/NetworkManager.in b/initscript/RedHat/NetworkManager.in
index e26e59e..fc51232 100755
--- a/initscript/RedHat/NetworkManager.in
+++ b/initscript/RedHat/NetworkManager.in
@@ -44,7 +44,7 @@ RETVAL=0
 start()
 {
 	echo -n $"Setting network parameters... "
-	sysctl -e -p /etc/sysctl.conf >/dev/null 2>&1
+	apply_sysctl >/dev/null 2>&1
 	success
 	echo
 
-- 
2.4.3

